axiom/otel/trace_integration_test.go

```diff
@@ -9,43 +7,83 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
 
 	"github.com/axiomhq/axiom-go/axiom"
 	axiotel "github.com/axiomhq/axiom-go/axiom/otel"
+	"github.com/axiomhq/axiom-go/internal/config"
+	"github.com/axiomhq/axiom-go/internal/test/testhelper"
 )
 
 func TestTracingIntegration(t *testing.T) {
-	ctx := context.Background()
+	cfg := config.Default()
+	if err := cfg.IncorporateEnvironment(); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Would it make sense to `require.NoError(t, cfg.IncorporateEnvironment())` here (and in similar places elsewhere)?